runelite-client/src/main/resources/open.osrs.properties

```diff
@@ -1,15 +1,15 @@
 runelite.version=@project.version@
 runescape.version=@rs.version@
-open.osrs.discord.appid=627741263881568257
```

Comment:
what even happened in this file

Reply:
> 
> 
> what even happened in this file

that wasnt supposed to be submitted, i thought i had it in my gitignore but i guess not, im absolutely garbage with version control

Reply:
Weird, still turn off after 1st phase for me